be/src/Project/Web/Seller/Seller.Web/Areas/Orders/ViewModel/OrderItemAttributeTableViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace Seller.Web.Areas.Orders.ViewModel
+{
+    public class OrderItemAttributeTableViewModel
```

Comment:
**Suggestion** (non-blocking): I think it should be `OrderItemAttributesTableViewMode`.